private/buf/bufprotopluginexec/version.go

```diff
@@ -120,6 +121,23 @@ func getFeatureProto3OptionalSupported(version *pluginpb.Version) bool {
 	return true
 }
 
+// Should I notify that I am OK with editions (and, if so, which ones)?
+func getFeatureEditionsSupported(version *pluginpb.Version) (supported bool, min, max descriptorpb.Edition) {
+	if version.GetMajor() > 5 || (version.GetMajor() == 5 && version.GetMinor() >= 27) {
+		// TODO: Update this to include later editions as they are supported in later versions of protoc.
+		return true, descriptorpb.Edition_EDITION_2023, descriptorpb.Edition_EDITION_2023
```

Comment:
Shouldn't the min edition be proto2 to match https://github.com/protocolbuffers/protobuf/blob/a978b75794a6ce4547c9db08a115c458d9190934/src/google/protobuf/compiler/cpp/generator.h#L77-L78?

Reply:
That is super weird. It is not necessary to set a min below 2023 because 2023 was the first edition. But I see that other plugins set it that way, too (even `protoc-gen-go`) 🤔 

I have to admit I find that odd since proto3 and proto2 were _not_ editions. Those entries in the enum, from my understanding, were just ways of attributing semantics of features for pre-editions syntaxes (in feature defaults) and not meant to be used this way.

Neither buf nor protoc actually need it to be set that way. For proto2 and proto3 files, these fields in the response are ignored:
https://github.com/protocolbuffers/protobuf/blob/main/src/google/protobuf/compiler/command_line_interface.cc#L2591-L2594
I can change it, for consistency with the original C++ impl, but TBH I find it counter-intuitive 🤷

Reply:
Updated